pbxplore/analysis/visualization.py

```diff
@@ -15,14 +15,14 @@
 import matplotlib.pyplot as plt
 
 try:
-    import weblogolib
+    import weblogo
```

Comment:
Does this change the version of weblogo that is required? If so, this should appear wherever we specify our dependencies (setup.py?)

Reply:
Yes, the change in the import name was made in Weblogo 3.7. I don't think it's necessary to specify it in our dependencies since it's the last version on pypi.

Reply:
It can be a problem for users who perform an upgrade rather than install from scratch. In that case, they may have PBxplore upgrading, but weblogo remaining at the previous version: then they would end up with a broken PBxplore and an unhelpful error message ("Cannot import weblogo" while the weblogo package is installed).

Reply:
Good point.

Reply:
Alright, then I will set the extra requirement as
```
# Extras requirements for optional dependencies
extras = {
    'analysis': ['weblogo>=3.7'],
    'all': ['weblogo>=3.7']
}
```

Reply:
sounds good